client/task-list/tasks/payments/RemotePayments/components/PaymentConnect.js

```diff
@@ -103,39 +140,49 @@ export const PaymentConnect = ( {
 		},
 	} );
 
-	return (
-		<Form
-			initialValues={ getInitialConfigValues() }
+	const DefaultSettings = ( props ) => (
+		<SettingsForm
+			fields={ fields }
+			isBusy={ isOptionsRequesting }
 			onSubmitCallback={ updateSettings }
+			onButtonClickCallback={ () => recordConnectStartEvent( key ) }
```

Comment:
Where does this connect to?  I'm not seeing this prop used anywhere.

Reply:
I'd forgotten to update this as well. It's actually the `onButtonClick` handler, fired when the submit button is clicked.